ui/Dockerfile

```diff
@@ -12,8 +12,10 @@ RUN apt-get update && \
     npm install && \
     npm install --global bower gulp-cli && \
     bower --allow-root install
-    
-    
+
+# Add environment variable values to env.js
+# ENV: apiUrl : backend url
+RUN sh env.sh 
```

Comment:
This won't work, this will run on the build time not on run time. Users who pull the docker image from the Docker hub won't be able to change the values when running.

Reply:
@agentmilindu Made a workaround, Please look at it.